frameworks/hdfs/universe/config.json

```diff
@@ -731,6 +736,11 @@
           "description": "Boolean which enables backend datanode-side support for the experimental DistributedFileSystem#getFileVBlockStorageLocations API.",
           "default": false
         },
+        "fsdataset_volume_choosing_policy": {
```

Comment:
Should we not set the default to the default values specified in the description?

Reply:
This setting doesn't have a default according to the official documentation: https://hadoop.apache.org/docs/stable/hadoop-project-dist/hadoop-hdfs/hdfs-default.xml

Reply:
Sorry, I thought a default was mentioned in the description?

Reply:
Yeah, it says the default is `org.apache.hadoop.hdfs.server.datanode.fsdataset.RoundRobinVolumeChoosingPolicy` but then doesn't actually put that in the value field in the chart. Anyhow, let's go with this as the default.